dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.33/V2_33_14__Add_jsonb_attributevalues_columns.sql

```diff
@@ -0,0 +1,30 @@
+alter table dataelement add column if not exists attributevalues  jsonb  default '{}'::jsonb;
+alter table dataset add column if not exists  attributevalues jsonb  default '{}'::jsonb;
```

Comment:
One space too much after exists in all statements.

Reply:
fixed